tox.ini

```diff
@@ -1,15 +1,19 @@
 # Generated from:
 # https://github.com/plone/meta/tree/master/config/default
-# Plus some manual settings: be careful not to lose those when reconfiguring!
+# See the inline comments on how to expand/tweak this configuration file
 [tox]
 # We need 4.4.0 for constrain_package_deps.
 min_version = 4.4.0
-envlist = py38,
-          py39,
-          py310,
-          py311,
-          pypy,
-          pypy3,
+envlist =
+    lint
+    test
+    dependencies
+    py38,
+    py39,
+    py310,
+    py311,
+    pypy,
```

Comment:
`pypy` could actually be removed: this would be for PyPy2, which we do not support.
`pypy3` should stay though.